src/renderer/model/db/clientBackend.ts

```diff
@@ -171,6 +171,7 @@ export class DBClientBackend implements CM.ClientBackend {
     }
   }
 
+  /* eslint-disable @typescript-eslint/ban-ts-ignore */
```

Comment:
Too broad, please use `eslint-disable-next-line` or reenable the rule after the function `createRequestGroup`.

Reply:
re-enabled after